backend/src/yeomanui.ts

```diff
@@ -274,7 +274,7 @@ export class YeomanUI {
 	  const generators: IQuestionsPrompt = await this.getGeneratorsPrompt();
       const response: any = await this.rpc.invoke("showPrompt", [generators.questions, "select_generator"]);
       this.replayUtils.clear();
-      SWA.updateGeneratorStarted(response.generator);
+      SWA.updateGeneratorStarted(response.generator, this.logger);
```

Comment:
didn't we already pass the logger once at --> SWA.createSWATracker(this.logger);

Reply:
I have changed swa logic because it does not work with websocket.
The SWA class should be rewritten, because it uses static methods. I did not wanted to do it in this change